provisioning/service/src/Config/BulkOperationMode.cs

```diff
@@ -2,22 +2,25 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System.Runtime.Serialization;
```

Comment:
Why is Serialization required? This is discouraged in .NET Core.

Reply:
We need for EnumMember